modules/ppcp-subscription/src/class-subscriptionmodule.php

```diff
@@ -80,7 +80,7 @@ function ( $description, $id ) use ( $container ) {
 				$settings                 = $container->get( 'wcgateway.settings' );
 				$subscription_helper      = $container->get( 'subscription.helper' );
 
-				return $this->display_saved_paypal_payments( $settings, $id, $payment_token_repository, $description, $subscription_helper );
+				return $this->display_saved_paypal_payments( $settings, $id, $payment_token_repository, (string) $description, $subscription_helper );
```

Comment:
I suggest casting to string also the second parameter, $id. Normally, it should be a string already, but as this value comes from the filter and can be modified by any other plugin, it's safer to cast it too.